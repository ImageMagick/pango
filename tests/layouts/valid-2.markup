
test the <i>blue</i> drink <u>after</u> dinner
