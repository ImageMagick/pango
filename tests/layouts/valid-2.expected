test the blue drink after dinner

---

wrapped: 0
ellipsized: 0
lines: 2

---

range 0 9
range 9 13
[9 13] style 2
range 13 20
range 20 25
[20 25] underline 1
range 25 2147483647

---

i=1, index=0, paragraph-start=1, dir=ltr 'test the blue drink after dinner
'
i=2, index=33, paragraph-start=1, dir=ltr ''

---

i=1, index=0, chars=9, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=c, 'test the '
i=2, index=9, chars=4, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=c, 'blue'
i=3, index=13, chars=7, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=c, ' drink '
i=4, index=20, chars=5, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=c, 'after'
  [20 25] underline 1
i=5, index=25, chars=7, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=c, ' dinner'
i=6, index=32, no run, line end
i=7, index=33, no run, line end
